lib/kitchen/driver/gce.rb

```diff
@@ -81,6 +81,7 @@ class Gce < Kitchen::Driver::Base
       default_config :use_private_ip, false
       default_config :wait_time, 600
       default_config :refresh_rate, 2
+      default_config :metadata, {}
```

Comment:
Can you update the README to include this new configuration option with an example? Maybe make a note that `created_by`, `test-kitchen-instance`, and `test-kitchen-user` are set automatically?

Reply:
Will do